examples/quickstart/main.rs

```diff
@@ -235,6 +235,26 @@ async fn main() {
             }
         };
 
+        // Drain any additional buffered messages to get the most recent one
```

Comment:
oh no no no no no, we can't do this in the normal quickstart!! Remember that for on chain protocols, we get deltas.. so we need to apply them all to the states, otherwise we will end up with weird states that do not correspond to the onchain state